exporter/prometheusexporter/accumulator.go

```diff
@@ -327,3 +334,81 @@ func copyMetricMetadata(metric pmetric.Metric) pmetric.Metric {
 
 	return m
 }
+
+func AccumulateNumericValues(prev, current, dest pmetric.NumberDataPoint) {
+	if current.StartTimestamp().AsTime().Before(prev.StartTimestamp().AsTime()) {
+		dest.SetStartTimestamp(current.StartTimestamp())
+	} else {
+		dest.SetStartTimestamp(prev.StartTimestamp())
+	}
+
+	older := prev
+	newer := current
+	if current.Timestamp().AsTime().Before(prev.Timestamp().AsTime()) {
+		older = current
+		newer = prev
+	}
+
+	newer.Attributes().CopyTo(dest.Attributes())
+	dest.SetTimestamp(newer.Timestamp())
+
+	switch newer.ValueType() {
+	case pmetric.NumberDataPointValueTypeDouble:
+		dest.SetDoubleValue(newer.DoubleValue() + older.DoubleValue())
+	case pmetric.NumberDataPointValueTypeInt:
+		dest.SetIntValue(newer.IntValue() + older.IntValue())
+	}
+}
+
+func AccumulateHistogramValues(prev, current, dest pmetric.HistogramDataPoint) {
+	if current.StartTimestamp().AsTime().Before(prev.StartTimestamp().AsTime()) {
+		dest.SetStartTimestamp(current.StartTimestamp())
+	} else {
+		dest.SetStartTimestamp(prev.StartTimestamp())
+	}
+
+	older := prev
+	newer := current
+	if current.Timestamp().AsTime().Before(prev.Timestamp().AsTime()) {
+		older = current
+		newer = prev
+	}
+
+	newer.Attributes().CopyTo(dest.Attributes())
+	dest.SetTimestamp(newer.Timestamp())
+
+	match := true
+	if older.ExplicitBounds().Len() == newer.ExplicitBounds().Len() {
+		for i := 0; i < newer.BucketCounts().Len(); i++ {
+			if older.ExplicitBounds().At(i) != newer.ExplicitBounds().At(i) {
+				match = false
+				break
+			}
+		}
+	} else {
+		match = false
+	}
+
+	if match {
+		dest.SetCount(newer.Count() + older.Count())
+		if newer.HasSum() && older.HasSum() {
+			dest.SetSum(newer.Sum() + older.Sum())
+		}
+
+		counts := make([]uint64, newer.BucketCounts().Len())
+		for i := 0; i < newer.BucketCounts().Len(); i++ {
+			counts[i] = newer.BucketCounts().At(i) + older.BucketCounts().At(i)
+		}
+		// dest.SetBucketCounts(pcommon.NewImmutableUInt64Slice(counts))
+	} else {
+		// use new value if bucket bounds do not match
+		dest.SetCount(newer.Count())
+		if newer.HasSum() {
+			dest.SetSum(newer.Sum())
+		}
+
+		// dest.SetBucketCounts(newer.BucketCounts())
+	}
+
+	// dest.SetExplicitBounds(newer.ExplicitBounds())
```

Comment:
```suggestion
	}
```

Don't leave commented code in the file.  Though it looks like this may be necessary.  If it is, uncomment it.  if not, remove it.